new/LIT7140Homily.xml

```diff
@@ -0,0 +1,72 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7140Homily" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Homily by Zacharias, bishop of Sakha, on the descent of Jesus, Mary, Joseph, and Salome to Egypt</title>
+                <title xml:lang="gez" xml:id="t2">ድርሳን፡ ዘአቡነ፡ አባ፡ ዘካርያስ፡ ኤጲስ፡ ቆጶስ፡ ትሩፍ፡ ዘለሀገረ፡ ሰኳ፡ መፍቀሪተ፡ ክርስቶስ፡ በእንተ፡ ርደቱ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ኀበ፡ ግብጽ ፡ ወእሙ፡
+                    እግዝእትነ፡ ማርያም፡ ወዮሴፍ፡ ወሰሎሜ፡ አመ፳ወ፩እምነ፡ ግንቦት፡</title>
+                <title xml:lang="gez" type="romanized" corresp="#t2">Dǝrsān za-ʾabuna ʾabbā Zakāryās ʾeṗṗis qoṗṗos tǝruf za-la-hagara Sakʷā mafqarita
+                    Krǝstos baʾǝnta rǝdatu la-ʾǝgziʾǝna ʾIyasus Krǝstos ḫaba Gǝbṣ wa-ʾǝmu ʾǝgzǝʾǝtǝna Māryām wa-Yosef wa-Salome ʾama 20wa4 ʾǝmǝnna Gǝnbot</title>
```

Comment:
If it is attested like this in Graf, I recommend to put t2 as an incipit in body. The information, that the feast is celebrated on 12th of Genbot can also be given in a short abstract.